### ISSUE TITLE
Documentation site is broken?

### ISSUE THREAD

**Issue by JoshLipps** [microsoft/angular-react#129]
https://microsoft.github.io/angular-react/ 

Just loads a white page... where are the docs hosted?

gh-pages job on circleCi seems like it has been failing since 9 months
ago (https://circleci.com/gh/microsoft/angular-react/138)

**Comment by aarongreenwald** [comment]
Hi @JoshLipps, 

Thanks for reporting this! You're right, the docs are down, and I'm working on getting them back up now. Will update when I get this worked out (they're up right now as I write this, but I'm still fiddling...will check-in again here when I am done). 

I appreciate the heads up.

**Comment by JoshLipps** [comment]
Thanks for the quick response!

**Comment by aarongreenwald** [comment]
Ok, the promised update:

Looks like the docs site was accidentally broken when we upgraded to angular 8 a few weeks ago. The confusing bit was that there was also a failing build in CircleCI on the gh-pages branch (for 9 months) but that build was unnecessary. It was triggered every time the docs changed and tried to build the project, which it couldn't, because the project doesn't exist on the gh-pages branch. 

I added a filter in the CI config so that CircleCI won't try to build the gh-pages branch. There's a build job on master that publishes the docs to the gh-pages branch, which is enough. 

As a side note...the docs can use some updating and polish. But that's a separate task for another time. 

Thanks again @JoshLipps for bringing this to my attention.
